src/assets/scripts/client/game/GameController.js

```diff
@@ -450,6 +472,23 @@ class GameController {
         this.game.last_score = this.game.score;
     }
 
+    /**
+     * @for GameController
+     * @method game_updateScoreHistory
+     * @param message {string}
+     */
+    game_updateScoreHistory(message, points) {
```

Comment:
I know a bunch of methods in here have this weird naming strategy. for this, though, let's remove `game_` from the name and go with just `updateScoreHistory`